From 1a44d2300402c5414d863bd141b872e0aee0a636 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Sun, 24 Mar 2013 16:16:42 +0100
Subject: [PATCH] OE: ubifs: ubifs-v2.6.24-silent-get-sb.patch

This patch came from OpenEmbedded but have no bearing on the
oxnas platform support.

The original patch wass applied by:

    git apply --whitespace=fix ubifs-v2.6.24-silent-get-sb.patch
    git add .

http://git.openembedded.org/openembedded/tree/recipes/linux/linux-2.6.24/ubifs-v2.6.24-silent-get-sb.patch

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 fs/ubifs/super.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/ubifs/super.c b/fs/ubifs/super.c
index 69429f0..7f46ff2 100644
--- a/fs/ubifs/super.c
+++ b/fs/ubifs/super.c
@@ -2020,8 +2020,8 @@ static int ubifs_get_sb(struct file_system_type *fs_type, int flags,
 	 */
 	ubi = open_ubi(name, UBI_READONLY);
 	if (IS_ERR(ubi)) {
-		ubifs_err("cannot open \"%s\", error %d",
-			  name, (int)PTR_ERR(ubi));
+		dbg_err("cannot open \"%s\", error %d",
+			name, (int)PTR_ERR(ubi));
 		return PTR_ERR(ubi);
 	}
 	ubi_get_volume_info(ubi, &vi);
-- 
1.7.1

